students/byeongyeon/westagram/users/views.py

```diff
@@ -1,3 +1,39 @@
-from django.shortcuts import render
-
 # Create your views here.
+import json
+
+from django.http            import JsonResponse
+from django.views           import View
+
+from users.models           import User
+from users.validator        import validate_email, validate_password, validate_mobile
+
+class RegisterView(View):      
+    def post(self, request):
+        data     = json.loads(request.body)
+        email    = data["email"]
+        password = data["password"]
+        mobile   = data["mobile"]
+
+        try:
+            if User.objects.filter(email=email).exists():
+                return JsonResponse({"message": "Email is already in use"}, status=400)
+            if not validate_email(email):
+                return JsonResponse({"message": "Email format is invalid"}, status=400)
+            if not validate_password(password):
+                return JsonResponse({"message": "Password format is invalid"}, status=400)
+            if not validate_mobile(mobile):
+                return JsonResponse({"message": "Mobile number format is invalid"}, status=400)
+
+            User.objects.create(
+                name     = data['name'],
+                email    = data['email'],
+                password = data['password'],
+                mobile   = data['mobile'],
+                address  = data['address'],
+                age      = data['age']
+            )
```

Comment:
위에서 선언한 변수를 활용하면 더 좋을 것 같습니다.
```suggestion
            User.objects.create(
                email    = email,
                password = password,
                mobile   = mobile,
                name     = data['name'],
                address  = data['address'],
                age      = data['age']
            )
```

Reply:
수정사항 반영하였습니다!